internals/secrethub/service_aws_init.go

```diff
@@ -88,7 +92,7 @@ func (cmd *ServiceAWSInitCommand) Register(r Registerer) {
 	clause.Flag("kms-key-id", "ID of the KMS-key to be used for encrypting the service's account key.").Required().StringVar(&cmd.kmsKeyID)
 	clause.Flag("role", "ARN of the IAM role that should have access to this service account.").Required().StringVar(&cmd.role)
 	clause.Flag("region", "The AWS region that should be used").StringVar(&cmd.region)
-	clause.Flag("desc", "A description for the service").StringVar(&cmd.description)
+	clause.Flag("desc", "A description for the service. Defaults to the ARN of the IAM role that is given access to the service.").StringVar(&cmd.description)
```

Comment:
Can we create an alias `--description` and `--descr` for this? I find myself typing it wrong every time? `--description` should be the default if you ask me.

Reply:
I'll create a ticket for this :+1:

Reply:
#140